dev-lisp/roswell/roswell-19.08.10.101.ebuild

```diff
@@ -22,13 +22,13 @@ src_prepare() {
 }
 
 src_compile() {
-	emake -j1
+	emake
```

Comment:
This should be the default function, so you can now remove this (and the phases below) altogether.